src/main/java/seedu/placebook/logic/commands/FindTagsCommand.java

```diff
@@ -30,7 +30,8 @@ public FindTagsCommand(PersonHasTagsPredicate predicate) {
     public CommandResult execute(Model model, Ui ui) {
         requireNonNull(model);
         model.updateFilteredPersonList(predicate);
-        model.updateState();
+        model.updateState(
+                String.format(Messages.MESSAGE_PERSONS_LISTED_OVERVIEW, model.getFilteredPersonList().size()));
         return new CommandResult(
```

Comment:
I took a few try. maybe for findTags, the message can be `undo command: findTags: 0 person listed`

Reply:
Oh ya the MESSAGE_SUCCESS of `findTag` command does not contain its COMMAND_WORD. I will fix this.